packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,210 @@
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+@breakpoints {
+  .rt-DataListRoot {
+    &:where(.rt-r-dl-orient-vertical) :where(.rt-DataListItem) {
+      display: flex;
+      flex-direction: column;
+      gap: var(--space-1);
+    }
+    &:where(.rt-r-dl-orient-horizontal) :where(.rt-DataListItem) {
+      display: grid;
+      /* Use subgrid so all the label columns remain aligned */
+      grid-template-columns: subgrid;
+      padding: 0;
+      gap: inherit;
+    }
+    &:where(.rt-r-dl-orient-vertical) :where(.rt-DataListLabel) {
+      white-space: normal;
+      padding: 0;
+      width: auto;
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * TRIM                                                                                            *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+@breakpoints {
+  .rt-DataListRoot {
+    &:where(.rt-r-dl-trim-start, .rt-r-dl-trim-both) :where(.rt-DataListItem:first-child) {
+      margin-top: calc(
+        var(--default-leading-trim-start) -
+          var(--line-height, calc(1em * var(--default-line-height))) / 2
+      );
+    }
+
+    &:where(.rt-r-dl-trim-end, .rt-r-dl-trim-both) :where(.rt-DataListItem:last-child) {
+      margin-bottom: calc(
+        var(--default-leading-trim-end) - var(--line-height, calc(1em * var(--default-line-height))) /
+          2
+      );
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * WIDTH                                                                                           *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-DataListLabel {
+  --dl-width-scale: 24;
+  width: var(--dl-label-width);
+  min-width: var(--dl-label-min-width);
+  max-width: var(--dl-label-max-width);
+}
+
+@breakpoints {
+  .rt-DataListLabel {
+    &:where(.rt-r-dl-width) {
+      /* Keep this selector here to maintain the source order */
+    }
+    &:where(.rt-r-dl-width-1) {
+      width: calc(var(--space-1) * var(--dl-width-scale));
+    }
+    &:where(.rt-r-dl-width-2) {
+      width: calc(var(--space-2) * var(--dl-width-scale));
+    }
+    &:where(.rt-r-dl-width-3) {
+      width: calc(var(--space-3) * var(--dl-width-scale));
+    }
```

Comment:
Using the base `space` values and a scale multiplier. Totally open to another approach but wanted the widths to stay connected to the base space system. If we like this we can also add more sizes if we think that would be helpful / meaningful. 

[See related comment.](https://github.com/radix-ui/themes/pull/237#discussion_r1458081752)

Reply:
I personally think tokens should remain consistent, ie. when I use a space value of 3 it should always be the same resulting px value.

Reply:
Let's reuse the existing width props as is because when it comes to using the space scale, "1", "2", etc. should always mean the same value, otherwise we'd invite trust issues around the prop values.

I do recognise this as an argument to review whether we need to expose space scale on all `width` props in other components at all. It's fair to say that in 90% of the cases the width values are too small to be useful on any component. Taking a note to sort it out.

Reply:
@vladmoroz I'm a bit confused because the largest width is ~64px that means majority of the width sizes wouldn't work for this use case. I get the need for consistency thought and perhaps most folks will just use the custom properties.

Reply:
Yeah fair. I think that here we'll end up doing custom properties only—just want to look at it altogether everywhere width is used because it seems to be the general case. Let's just ignore the issue in this PR and follow up on the width prop after